types.go

```diff
@@ -0,0 +1,12 @@
+package main
+
+type KeyModel struct {
```

Comment:
Let's throw this in a `model` directory with a `model` package name. This will allow us to name the struct just `Key`. When we instantiate new keys we will use `model.Key` as opposed to `model.KeyModel`.

Reply:
Done!